python/ray/util/accelerators/accelerators.py

```diff
@@ -8,6 +8,13 @@
 INTEL_MAX_1550 = "Intel-GPU-Max-1550"
 INTEL_MAX_1100 = "Intel-GPU-Max-1100"
 INTEL_GAUDI = "Intel-GAUDI"
+AMD_INSTINCT_MI100 = "AMD-Instinct-MI100"
+AMD_INSTINCT_MI250x = "AMD-Instinct-MI250X"
+AMD_INSTINCT_MI250 = "AMD-Instinct-MI250X-MI250"
```

Comment:
should it be `AMD_INSTINCT_MI250X_MI250`?

Reply:
This is the correct way to identify this product as per our internal documentation

Reply:
I mean should the variable name match the actual string?

Reply:
No I think we can leave it as is